### PR TITLE
mmpycocotools

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
run tests

**Comment by Ilya-Krylov** [comment]
@druzhkov-paul e.g. in your yesterday PR : "Building wheel for mmpycocotools (setup.py): finished with status 'error'"
http://134.191.240.124/job/ote/job/precommit/job/mmdetection/157/consoleFull

File: docker/Dockerfile.cpu [hunk 1]
```diff
@@ -34,7 +34,6 @@ RUN git clone --recursive --branch $MMDET_BRANCH https://github.com/opencv/mmdet
 WORKDIR /mmdetection
```
Comment (druzhkov-paul) [inline docker/Dockerfile.cpu:34]: Could you also change `MMDET_BRANCH` a few lines above?

**Review by druzhkov-paul — APPROVED** [review]

File: docker/Dockerfile.cpu (hunk 1 above)
Comment (Ilya-Krylov) [reply docker/Dockerfile.cpu:34]: done

**Comment by Ilya-Krylov** [comment]
run tests

**Pull request merged**